packages/components/src/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { HTMLAttributes } from "react"
+import { FormattedMessage, useIntl } from "@cultureamp/i18n-react-intl"
+import classNames from "classnames"
+import { BrandMomentError } from "@kaizen/draft-illustration"
+import { Paragraph } from "@kaizen/typography"
+import { BrandMoment } from "~components/BrandMoment"
+import { ArrowRightIcon } from "~components/SVG/icons/ArrowRightIcon"
+import { EmailIcon } from "~components/SVG/icons/EmailIcon"
+import { OverrideClassName } from "~types/OverrideClassName"
+import { useMessages } from "./hooks"
+import styles from "./ErrorPage.module.scss"
+
+const getMailToHref = (code: number): string => {
+  const supportEmail = "support@cultureamp.com"
+  const subject = "Houston we have a problem"
+  const body = `Hi there,\n\nI received a ${code} error page while I was trying to...`
+  return encodeURI(`mailto:${supportEmail}?subject=${subject}&amp;body=${body}`)
+}
+
+const HOME_HREF = "/app/home"
+
+export type ErrorPageProps = {
+  code: number
+  title?: string
+  message?: React.ReactNode | string
+  callToAction?: {
+    onContactSupport: () => void
+    homeHref?: string
+  }
+} & OverrideClassName<HTMLAttributes<HTMLDivElement>>
+
+export const ErrorPage = ({
+  code,
+  title,
+  message,
+  callToAction,
+  classNameOverride,
+}: ErrorPageProps): JSX.Element => {
+  const { formatMessage } = useIntl()
+  const translations = useMessages(code)
```

Comment:
I think the naming of `useMessages` is too close to the `i18n-react-intl` package, and I feel like using the word `content` here better describes what it's getting used as.

```suggestion
  const content = useErrorMessages(code)
```

Reply:
done